core/src/main/scala/org/http4s/Headers.scala

```diff
@@ -78,7 +78,7 @@ final class Headers private (private val headers: List[Header]) extends AnyVal {
       this.headers.foreach { orig =>
         orig.parsed match {
           case _: Header.Recurring => acc += orig
-          case _: `Set-Cookie` => acc += orig
+          // case _: `Set-Cookie` => acc += orig
```

Comment:
This is a special case in the spec and needed to be a special case in the old design, but it is covered by the new one.

Reply:
To be clear, no further action needed on this one.  We'll hopefully be removing this class entirely soon.